benchmarking/cluster3nodes/run-3node-cluster.sh

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
```

Comment:
I'm a little afraid this is going to break as the configuration changes happen to the node..

If we shared more with `scripts/` (which even has `lib-node.sh` for this purpose), I'd say we might have better chances of surviving these changes..

Reply:
I wanted to keep this separate so we can make changes/adaptations without interfering with other uses. IMO, the scripts will not be used in the future.